tests/src/application/controllers/forget-password.spec.ts

```diff
@@ -23,6 +25,14 @@ const makeForgetPassword = (): ForgetPassword => {
   return new ForgetPasswordStub()
 }
 
+const makeFakeRequest = (): HttpRequest => {
```

Comment:
Could you better express what controller this request is about and add the fakes folder. If your branch is outdated and doesn't have the "mocks" and "helpers" folder, disregard it.